.travis.yml

```diff
@@ -13,14 +13,14 @@ before_install:
   - pip install --upgrade pip setuptools wheel
 
 install:
-  - pip install --upgrade pycodestyle
-  - pip install --upgrade pyflakes
+  - pip install --upgrade flake8
 
 before_script:
   - if [[ $TRAVIS_PYTHON_VERSION == 3* ]]; then 2to3 -n -w --no-diffs troposphere scripts/cfn2py scripts/gen.py examples/CustomResource.py tests; fi
-  - pycodestyle --version
+  - flake8 --version
   - pycodestyle --show-source --show-pep8 .
   - pyflakes .
+  - flake8 . --show-source 
```

Comment:
The executions of `pycodestyle` and `pyflakes` can be removed, right?

Reply:
Yes.